addons/web/static/src/core/utils/draggable_hook_builder.js

```diff
@@ -64,6 +65,7 @@ import { hasTouch, isBrowserFirefox, isIOS } from "../browser/feature_detection"
  * @property {boolean} [enabled=true]
  * @property {number} [speed=10]
  * @property {number} [threshold=20]
+ * @property {"horizontal"|"vertical"} [direction]
```

Comment:
I think I would have chosen "x" and "y" instead

Reply:
no test?

Reply:
not sure to understand why you need it btw

Reply:
> not sure to understand why you need it btw

it is better to prevent vertical scrolling when resizing a pill or selecting cells on a given line